Lib/test/test_http_cookiejar.py

```diff
@@ -313,6 +314,32 @@ def _interact(cookiejar, url, set_cookie_hdrs, hdr_name):
 
 
 class FileCookieJarTests(unittest.TestCase):
+    def test_constructor_with_str(self):
+        filename = test.support.TESTFN
+        c = LWPCookieJar(filename)
+        self.assertEqual(c.filename, filename)
+        self.assertIsInstance(c.filename, str)
```

Comment:
Are the type checks necessary? Unless the docs explicitly state that then I don't think they are necessary.

Reply:
no, they are not necessary, I have removed them. Thank you for your review.